orc8r/cloud/go/services/certifier/protos/certifier.proto

```diff
@@ -76,7 +76,13 @@ message Resource {
   Effect effect = 1;
   Action action = 2;
   ResourceType resourceType = 3;
+  // Resource is the path of the resource
   string resource = 4;
+  // Resource ID is an optional field for tenant and network resources
+  oneof resource_id {
+    string tenant_id = 5;
+    string network_id = 6;
+  }
```

Comment:
Although these seem a bit redundant, I did this because network/tenant resources should match for both their network/tenant ID resource policies and their URL types. I figured if we could parse the resource out at the rest middleware stage, we wouldn’t have to do the thing where we had to semantically understand each path to get the resource id.